de.js

```diff
@@ -0,0 +1,334 @@
+    enter_seed_phrase: `Seed-Phrase eingeben`,
+    enter_seed_phrase_instructions: `Gib deine 12-Wort-Seed-Phrase ein um deine bestehende Wallet wiederherzustellen`,
+    paste: `Einfügen`,
+    restore_wallet: `Wallet wiederherstellen`,
+    wallet_imported: `Wallet importiert!`,
+    wallet_imported_body: `Deine Wallet ist einsatzbereit.`,
+    restore_existing_wallet: `Deine bestehende Wallet wiederherstellen`,
+    restore_existing_wallet_instructions: ` Du hast also bereits eine Wallet? Dann lass sie uns wiederherstellen!`,
+    restore_from: `Wiederherstellen per`,
+    restore_from_cloud_description: `Wiederherstellen deiner Wallet per`,
+    restore_with_seed_phrase: `Wiederherstellen mit einer Seed-Phrase`,
+    restore_with_seed_phrase_description: `Nutze deine Seed-Phrase um eine bestehende MyDoge-Wallet wiederherzustellen`,
+    no_cloud_backup: `Kein Cloud-Backup`,
+    no_cloud_backup_body: `Kein Cloud-Backup gefunden. Bitte per Seed-Phrase wiederherstellen. Oder erstelle eine neue Wallet`,
+    create_wallet: `Eine Wallet erstellen`,
+    already_have_wallet: ` Ich habe bereits eine Wallet`,
+    buy_send_receive_dogecoin: `Kaufe, Sende & Empfange Dogecoin`,
+    send_doge_tips: `Versende Doge-Tips auf 𝕏, verwahre deine Doge sicher und sei deine eigene Bank`,
+    shop_gift_cards_apparel: `Kaufe Geschenkkarten und Kleidung`,
+    buy_gift_cards: `Kaufe Geschenkkarten mit Dogecoin oder shoppe die neueste Doge-Mode`,
+    enter_the_dogeverse: `Betritt das Dogeverse`,
+    claim_dogtag: `Sichere dir dein DogeTag, finde Freunde, erhalte Zahlungen über deinen Link und teile Memes`,
+    so_much_tip: `So Tip, Much Wow!`, //leave it in english I`d say^^ oder `Wow, soviel Trinkgeld`
+    success: `Erfolg`,
+    sent_doge: `Du hast`, // These 2 lines belong together?
+    some_doge: `ein paar Doge gesendet!`, // OK if nowhere else... else: sent_doge: `Du sendetest`, ... some_doge: `ein paar Doge`, but sounds/looks terrible
+    total: `Gesamt`,
+    done: `Erledigt`,
+    shop_with_doge: `Shoppe mit Doge`,
+    oops: `Hoppla!`,
+    no_comments_yet: `Noch keine Kommentare`,
+    to_post_a_comment: `einen Kommentar abgeben`,
+    your_comment: `Dein Kommentar`,
+    post: `Senden`,
+    delete: `Löschen`,
+    enable_push_notifications: `Push-Benachrichtigungen aktivieren?`,
+    get_notifications_for_tips: `Benachrichtigungen für Trinkgelder und Nachrichten erhalten`,
+    no_thanks: `Nein, danke`,
+    open_settings: `Einstellungen öffnen`,
+    swipe_for_more: `Für mehr, wische nach oben`,
+    transaction_confirmation_alert: `Überprüfe immer zweimal an wen du etwas sendest bevor du bestätigst. Sende niemals an eine Auszahlungsadresse einer Börse. Dogecoin-Transaktionen können nicht rückgängig gemacht werden!`,
+    clear_amount: `Betrag löschen`,
+    set_amount: `Betrag bestimmen`,
+    save_qr: `QR speichern`,
+    WALLET_ADDRESS: `WALLET-ADRESSE`,
+    dogetag_link: `DOGETAG-LINK`,
```

Comment:
See Line 211: DogeTag is similar to Username so maybe here just 'WEB-LINK' or in Line 211 and here 'DOGETAG-WEB-LINK'

```suggestion
    dogetag_link: `WEB-LINK`,
```
or
```suggestion
    dogetag_link: `DOGETAG-WEB-LINK`,
```

Reply:
We want to call it dogetag